src/ripple/ledger/impl/View.cpp

```diff
@@ -687,107 +687,114 @@ dirNext (ApplyView& view,
     return true;
 }
 
-std::function<void (SLE::ref, bool)>
+std::function<void (SLE::ref)>
 describeOwnerDir(AccountID const& account)
 {
-    return [account](std::shared_ptr<SLE> const& sle, bool)
+    return [&account](std::shared_ptr<SLE> const& sle)
```

Comment:
I haven't audited the calls of `describeOwnerDir`, but capturing by value here is much safer. It's not obvious that the lifetime of the `account` parameter is longer than the lifetime of the return value. And even if we do now, this seem fragile for a small optimization.